include/ceed-backend.h

```diff
@@ -65,6 +65,11 @@ CEED_EXTERN int CeedGetObjectDelegate(Ceed ceed, Ceed *delegate,
                                       const char *objname);
 CEED_EXTERN int CeedSetObjectDelegate(Ceed ceed, Ceed delegate,
                                       const char *objname);
+CEED_EXTERN int CeedGetOperatorFallbackResource(Ceed ceed,
+    const char **resource);
+CEED_EXTERN int CeedSetOperatorFallbackResource(Ceed ceed,
+    const char *resource);
+CEED_EXTERN int CeedGetOperatorFallbackParentCeed(Ceed ceed, Ceed *parent);
```

Comment:
Are these necessarily `Operator` fallbacks, or does `CeedGetFallbackResource` make sense?

Reply:
Currently only Operators need this fallback idea, but I think that we could more generally use this fallback idea in the future.

Reply:
Updated